litestar/contrib/minijinja.py

```diff
@@ -172,7 +172,7 @@ def register_template_callable(
         Returns:
             None
         """
-        self.engine.add_global(key, pass_state(template_callable))
+        self.engine.add_global(key, pass_state(_transform_state(template_callable)))
```

Comment:
I needed to do this unrelated change in order to be ble to access the state while using this engine